roles/openshift_logging/tasks/main.yaml

```diff
@@ -3,11 +3,17 @@
     msg: Only one Fluentd nodeselector key pair should be provided
   when: "{{ openshift_logging_fluentd_nodeselector.keys() | count }} > 1"
 
-- name: Set default image variables based on deployment_type
-  include_vars: "{{ item }}"
-  with_first_found:
-    - "{{ openshift_deployment_type | default(deployment_type) }}.yml"
-    - "default_images.yml"
+- name: Set image config defaults
+  set_fact:
+    logging_image_prefix: "{{ openshift_logging_image_prefix | openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
```

Comment:
this seems like it could work. I was thinking more that since we'd keep the first found but use the 'private variable' idea where we either use `openshift_logging_image_prefix` if its provided otherwise we use `__openshift_logging_image_prefix`